en/docs/administer/key-managers/configure-wso2is-connector.md

```diff
@@ -280,7 +281,7 @@ Follow the steps given below to configure WSO2 IS as a third-party Key Manager c
       <tr class="even">
       <td><b>Token Handling Options</b></td>
       <td>This provides a way to validate the token for this particular authorization server.
-      This is mandatory if the Token Validation Method is <b>introspect</b></td>
+      This is mandatory if the Token Validation Method is **introspect**</td>
```

Comment:
Why have you changed the HTML code used for bold to markdown in the middle of HTML tags?